.gitignore

```diff
@@ -80,6 +80,7 @@ scikits
 !np_datetime_strings.c
 *.c
 *.cpp
+*.pxi
```

Comment:
further we DON't want to include an auto-generated file in git. So yes maybe should specifically exclude them in .gitignore (or rather the auto-gen ones)